packages/solid/store/src/server.ts

```diff
@@ -103,6 +117,10 @@ export function createMutable<T>(state: T | Store<T>): T {
   return state as T;
 }
 
+export function modifyMutable<T>(state: T, modifier: (state: T) => T) {
+  notSup();
+}
```

Comment:
Should this throw? Or should it be a no-op? I think that a no-op could be misleading, f.e. someone would think it worked, then log values only to see they didn't change.

Reply:
To be fair a lot of these methods still work on the server but just don't do anything. Like modifyMutable being missing is an oversight and should just function the same as in the client.